elifetools/tests/fixtures/test_body_block_content/content_41_expected.py

```diff
@@ -0,0 +1,5 @@
+from collections import OrderedDict
+expected = OrderedDict([
+    ('type', 'paragraph'),
+    ('text', 'Text <i>more</i> text ')
```

Comment:
would have expected `and a comment` somewhere?

Reply:
When I reviewed the logic, it is set to strip out all comments when rendering to HTML, so that is maintained here. They could possibly pass through as comments, but it was not what we do right now.

Reply:
For reference, it happens here https://github.com/elifesciences/elife-tools/blob/develop/elifetools/utils_html.py#L10